src/openrct2-ui/ride/Construction.cpp

```diff
@@ -24,6 +24,209 @@ using namespace OpenRCT2::TrackMetaData;
+        TrackElemType::LeftQuarterBankedHelixLargeUp,
+        TrackElemType::RightQuarterBankedHelixLargeUp,
+        TrackElemType::LeftQuarterBankedHelixLargeDown,
+        TrackElemType::RightQuarterBankedHelixLargeDown,
+        TrackElemType::LeftQuarterHelixLargeUp,
+        TrackElemType::RightQuarterHelixLargeUp,
+        TrackElemType::LeftQuarterHelixLargeDown,
+        TrackElemType::RightQuarterHelixLargeDown,
+        kSeparator,
+
+        // (Wooden) water splash
+        TrackElemType::Watersplash,
+        kSeparator,
+
+        // River Rapids
+        TrackElemType::Waterfall,
+        TrackElemType::Rapids, // Also used for Monster Trucks
+        TrackElemType::Whirlpool,
+        kSeparator,
+
+        // Spinning tunnel
+        TrackElemType::SpinningTunnel,
+        kSeparator,
+
+        // Reverser pieces
+        TrackElemType::LeftReverser,
+        TrackElemType::RightReverser,
+        TrackElemType::LogFlumeReverser,
+        kSeparator,
+
+        // Reverse freefall pieces
+        TrackElemType::ReverseFreefallSlope,
+        TrackElemType::ReverseFreefallVertical,
+        kSeparator,
+
+        // Air thrust pieces
+        TrackElemType::AirThrustTopCap,
+        TrackElemType::AirThrustVerticalDown,
+        TrackElemType::AirThrustVerticalDownToLevel,
+        kSeparator,
+
+        // Loops
+        TrackElemType::LeftVerticalLoop,
+        TrackElemType::RightVerticalLoop,
+        TrackElemType::HalfLoopUp,
+        TrackElemType::HalfLoopDown,
+        TrackElemType::FlyerHalfLoopUninvertedUp,
+        TrackElemType::FlyerHalfLoopInvertedDown,
+        TrackElemType::FlyerHalfLoopInvertedUp,
+        TrackElemType::FlyerHalfLoopUninvertedDown,
+        TrackElemType::LeftMediumHalfLoopUp,
+        TrackElemType::LeftMediumHalfLoopDown,
+        TrackElemType::RightMediumHalfLoopUp,
+        TrackElemType::RightMediumHalfLoopDown,
+        TrackElemType::LeftLargeHalfLoopUp,
+        TrackElemType::LeftLargeHalfLoopDown,
+        TrackElemType::RightLargeHalfLoopUp,
+        TrackElemType::RightLargeHalfLoopDown,
+        TrackElemType::LeftFlyerLargeHalfLoopUninvertedUp,
+        TrackElemType::LeftFlyerLargeHalfLoopInvertedDown,
+        TrackElemType::LeftFlyerLargeHalfLoopInvertedUp,
+        TrackElemType::LeftFlyerLargeHalfLoopUninvertedDown,
+        TrackElemType::RightFlyerLargeHalfLoopUninvertedUp,
+        TrackElemType::RightFlyerLargeHalfLoopInvertedDown,
+        TrackElemType::RightFlyerLargeHalfLoopInvertedUp,
+        TrackElemType::RightFlyerLargeHalfLoopUninvertedDown,
+        TrackElemType::MultiDimInvertedFlatToDown90QuarterLoop,
+        TrackElemType::Up90ToInvertedFlatQuarterLoop,
+        TrackElemType::InvertedFlatToDown90QuarterLoop,
+        TrackElemType::MultiDimUp90ToInvertedFlatQuarterLoop,
+        TrackElemType::MultiDimFlatToDown90QuarterLoop,
+        TrackElemType::MultiDimInvertedUp90ToFlatQuarterLoop,
+        kSeparator,
```

Comment:
I think the current order of inversions could be better, and I have come up with a few alternate orders. I have not yet decided which of these one is best:
"element size descending"
- Loops
- Zero-G/Dive Loop
- Corkscrew
- Barrel Roll
- Inline Twist
"element size ascending"
- Inline Twist
- Barrel Roll
- Corkscrew
- Zero-G/Dive Loop
- Loops
"element topology descending"
- Loops
- Corkscrew
- Zero-G/Dive Loop
- Barrel Roll
- Inline Twist
"element topology ascending"
- Inline Twist
- Barrel Roll
- Zero-G/Dive Loop
- Corkscrew
- Loops

I put these here if anyone has a strong preference. I am slightly partial to "element size descending"